docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+# Load data from Amazon S3
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load (`WITH BROKER`)
+2. Synchronous loading using the `FILES()` table function
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
```

Comment:
The figure needs to be updated (HDFS or cloud storage -> AWS S3). I'll work on it after the doc is done, 'cause I have the editable version of the figure.

Reply:
I think it is OK to leave general for two reasons:
- the image is still correct
- the reader will see this and realize that they have choices (S3, other cloud providers, and HDFS)

This is also something that I would move into an include file and source into the other docs (MinIO, GCS,  etc.)

Reply:
Got it! Then I will cross the figure update thing out from my task list!